src/components/ItemsList/ItemsList.tsx

```diff
@@ -0,0 +1,19 @@
+import { getVisibleItems } from '../../utils';
+
+type Props = {
+  items: string[],
+  firstElementIndex: number,
+  lastElementIndex: number,
+};
+
+export const ItemsList: React.FC<Props> = ({
```

Comment:
It's up to you, but I geuss it's a good practice to name Props type as componentNameProps -> ItemsListProps

btw, Items speaks for itself, it has letter 's' in the end, so no need to add word List in the name of the component